serving/src/main/java/feast/serving/service/HistoricalServingService.java

```diff
@@ -50,18 +50,17 @@ public HistoricalServingService(
   @Override
   public GetFeastServingInfoResponse getFeastServingInfo(
       GetFeastServingInfoRequest getFeastServingInfoRequest) {
-    try {
+    if (retriever instanceof BigQueryHistoricalRetriever) {
```

Comment:
I might have not been clear when I was referring to the try/catch. What I mean is, can we avoid hardcoding specific retriever implementations within `HistoricalServingService.java`?